ui/components/ui/index.ts

```diff
@@ -7,6 +7,7 @@ export * from "./chart/Chart";
 export * from "./content-layout/content-layout";
 export * from "./dialog/dialog";
 export * from "./download-icon-button/download-icon-button";
+export * from "./download-threatscore-button/download-threascore-button";
```

Comment:
The `UI` folder is for generic components. Components with a well-defined scope should go inside their assigned task folder.  

In this case, for example, it should go under `ui/components/compliances`

I’m happy to give you more details if anything is unclear

Reply:
totally agree, moved